src/status_im/ui/components/qr_code_viewer/views.cljs

```diff
@@ -15,8 +16,12 @@
 
 (defn qr-code-viewer [{:keys [style hint-style footer-style footer-button value hint legend]}]
   (if value
-    (let [{:keys [width]} @(re-frame/subscribe [:dimensions/window])]
+    (let [{:keys [width]} @(re-frame/subscribe [:dimensions/window])
+          snt-amount @(re-frame/subscribe [:get-in [:my-profile/tribute-to-talk :snt-amount]])]
```

Comment:
better to use `defview` macro

Reply:
I don't think it would play nice with conditional subscriptions based on value from params because the subs are done outside of the fn in defview, so it would never update after first rendering

Reply:
what is conditional subscriptions ? value condition must be moved inside letsubs